src/android/OpenTokAndroidPlugin.java

```diff
@@ -588,18 +589,9 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
                 return true;
             }
         } else if (action.equals("initSession")) {
-            apiKey = args.getString(0);
```

Comment:
With the new changes, from the execute method we are utilizing the token, apiKey and sessionId and passing this to VonageAcitivty where all the calls and there handling will take place.